get_sbu_data.py

```diff
@@ -0,0 +1,167 @@
+#fac = get_echo_data(sql)
+"""Note: because of errors in the EPA's own data, the above query may not always return accurate results.
+For instance, sometimes county names are listed as just "Tompkins"; sometimes they are listed as "Tompkins County".
+A more robust version utilizes another approach. See below:
+"""
+# Get county names based on Steve's work. If we know our county of interest is Tompkins, NY, we can search for other names it may be listed under
+counties = pd.read_csv("https://raw.githubusercontent.com/edgi-govdata-archiving/ECHO_modules/main/data/state_counties_corrected.csv") # Get a county name lookup
+counties = counties.groupby(by=["FAC_STATE", "County", "FAC_COUNTY"]).count() # Restructure the data
+counties = counties.iloc[(counties.index.get_level_values('County') == selected_county) & (counties.index.get_level_values('FAC_STATE') == selected_state)].reset_index() # Search for COUNTY, STATE
+counties = list(counties["FAC_COUNTY"].unique()) # List FAC_COUNTY names in ECHO associated with COUNTY
+# Format a sql query
+c = "("
+for county in counties:
+  c += "\'"+county+"\',"
+c = c[:-1] + ")"
+sql = 'select * from "ECHO_EXPORTER" where "FAC_COUNTY" in ' + c + ' and "FAC_STATE" = \'' + selected_state + '\'' # "ECHO_EXPORTER" contains basic info about all regulated industrial facilities, including location
+print(sql)
+fac = get_echo_data(sql)
+# display facilities in county
+fac
+
+
+st.header('Get County Boundaries')
+# This one gets a little complicated because of how the data are structured and the fact that a county name may not be unique (e.g. there are multiple Cedar Counties in the US
+statefp = fips[selected_state] # Get state's code
+sql = 'select * from "tl_2020_us_county" where "name" = \''+county_name +'\' and "statefp" = \''+str(statefp)+'\'' # Select specific county depending on state code
+county, state = get_spatial_data("County", [selected_state], spatial_tables, fips, county_name)
+# display county boundaries
+county
+
+
+def marker_text( row, no_text ):
+    '''
+    Create a string with information about the facility or program instance.
+
+    Parameters
+    ----------
+    row : Series
+        Expected to contain FAC_NAME and DFR_URL fields from ECHO_EXPORTER
+    no_text : Boolean
+        If True, don't put any text with the markers, which reduces chance of errors 
+
+    Returns
+    -------
+    str
+        The text to attach to the marker
+    '''
+
+    text = ""
+    if ( no_text ):
+        return text
+    if ( type( row['FAC_NAME'] == str )) :
+        try:
+            text = row["FAC_NAME"] + ' - '
+        except TypeError:
+            print( "A facility was found without a name. ")
+        if 'DFR_URL' in row:
+            text += " - <p><a href='"+row["DFR_URL"]
+            text += "' target='_blank'>Link to ECHO detailed report</a></p>" 
+    return text
+
+def bivariate_map(regions, points, bounds=None, no_text=False):
+    '''
```

Comment:
There is an inconsistent formatting for the function specs. Try to keep them organized the same way in order for other coders to understand what your function does.